.changeset/silver-dolphins-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": minor
+---
+
+crud generator: add support for enum arrays in input
```

Comment:
IMO we should start being consistent with the name of the generator. API Generator, API CRUD Generator, CRUD Generator, CRUD Generator (API)... What do you prefer?

Reply:
the changeset is in the context of the api package, that's why I did leave out the "api" (yes, I did actually think about that)